### PR TITLE
Limit Switches

### PR DESCRIPTION
**Pull request by vladaviedov**
Added all the limit switch logic.
The E-Board switch works, the back (magnetic) switch has not been setup correctly yet, but it did work.

### CODE REVIEW

**Comment by vladaviedov** [comment]
Magnetic switch now works

File: src/main/java/frc/robot/subsystems/LiftRotater.java [hunk 1]
```diff
@@ -9,38 +10,63 @@
...
-        robotLiftRotateMotorLeft.set(v);
-        robotLiftRotateMotorRight.set(-v);
+        // System.out.println("LIMIT SWITCH 1: " + forwardLimit.get());
+        // System.out.println("LIMIT SWITCH 2: " + backwardLimit.get());
+
+        // System.out.println("left encoder: " + robotLiftRotaterLeftEncoder.getPosition());
+
+        // Positive v = moving towards E-Board
+        // Negative v = moving towards Acquisition
+        if ((v > 0 && forwardLimit.get()) || (v < 0 && backwardLimit.get())) {
```
Comment (patatoes1235) [inline src/main/java/frc/robot/subsystems/LiftRotater.java:64]: I don't understand why this changed, but it looks like it works

**Review by patatoes1235 — APPROVED** [review]
This should work, i think

File: src/main/java/frc/robot/subsystems/LiftRotater.java (hunk 1 above)
Comment (vladaviedov) [reply src/main/java/frc/robot/subsystems/LiftRotater.java:64]: The old code would stop the lift from moving entirely when either of the limit switches were pressed.
We just want it to stop moving in one direction.

Comment (patatoes1235) [reply src/main/java/frc/robot/subsystems/LiftRotater.java:64]: Ohhh yea I’m just dumb ok

**Pull request merged**